encord/user_client.py

```diff
@@ -113,7 +113,15 @@ def get_project(self, project_hash: str) -> Project:
         config = SshConfig(self.user_config, resource_type=TYPE_PROJECT, resource_id=project_hash)
         querier = Querier(config)
         client = EncordClientProject(querier=querier, config=config)
-        return Project(client)
+
+        orm_project = client.get_project()
+
+        # TODO: it seems like client.get_ontology should be used here, but this leads to the permission errors
+        ontology_hash = orm_project["ontology_hash"]
```

Comment:
To avoid the permission errors, I believe that from within the `Project` we can request the ontology structure. 

Maybe we should just add the ontology structure directly into the project (maybe even within the `Project`'s `init` method).

The reasoning goes as following:
* Right now just because you have access to a project, you don't have full access to an ontology (e.g. you don't necessarily have access to change the ontology title) -> it would be better if we're explicit that the project only gets the ontology structure at this point.